### PR TITLE
remove opcomm

### PR DESCRIPTION
**Pull request by SpencerCreveling**
Check one:
- [x] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):
my first commit as a constitutional maintainer

### CODE REVIEW

**Comment by galenguyer** [comment]
I'm Galen and I approve this message

**Review by galenguyer — APPROVED** [review]

**Review by Mstrodl — APPROVED** [review]

**Comment by geoffmoss0** [comment]
"Add more commits by pushing to the `free-Klaus` branch"

File: constitution.tex [hunk 1]
```diff
@@ -477,17 +469,15 @@
...
+House History & 15\% \\
 \hline
-House Improvements & 15\% \\
+House Improvements & 20\% \\
 \hline
-Research and Development & 20\% \\
+Research and Development & 25\% \\
 \hline
-Social & 20\% \\
+Social & 25\% \\
```
Comment (JimDaGuy) [inline constitution.tex:480]: Can't conscionably advocate for increasing the social budget. As we all know, social bad.

Comment (geoffmoss0) [reply constitution.tex:480]: I think "Evil Board" (EBoard for short) would be more concise here

Comment (crigney3) [reply constitution.tex:480]: @SpencerCreveling Before I bring this up at house meeting, I think you should add a line that defines EBoard as "Evil Board", preferably in as many commits as possible

**Pull request closed without merging**